core/src/main/resources/oracle/weblogic/deploy/aliases/category_modules/ServerTemplate.json

```diff
@@ -397,7 +397,22 @@
             "wlst_type": "ServerDebug",
             "default_name_value": "${NO_NAME_0:%SERVERTEMPLATE%}",
             "version": "[12.1.2,)",
-            "folders": {},
+            "folders": {
+                "DebugScope": {
+                    "wlst_type": "DebugScope${:s}",
+                    "child_folders_type": "multiple",
+                    "wlst_mode": "offline",
```

Comment:
Why online is not supported?

Reply:
I cannot create online, and also cannot cd to one I created in offline in WLST.

Reply:
wls:/externalstage/edit/Servers/AdminServer/ServerDebug/AdminServer/DebugScopes !> create('weblogic.connector','DebugScopes')
MBean type DebugScope with name weblogic.connector has been created successfully.
wls:/externalstage/edit/Servers/AdminServer/ServerDebug/AdminServer/DebugScopes !> cd('weblogic.connector')
wls:/externalstage/edit/Servers/AdminServer/ServerDebug/AdminServer/DebugScopes/weblogic.connector !> ls()

-r--   DynamicallyCreated                           false
-rw-   Enabled                                      false
-r--   Id                                           0
-rw-   Name                                         weblogic.connector
-rw-   Notes                                        null
-rw-   Tags                                         null
-r--   Type                                         DebugScope

-r-x   addTag                                       Boolean : String(tag)
-r-x   freezeCurrentValue                           Void : String(attributeName)
-r-x   getInheritedProperties                       String[] : String[](propertyNames)
-r-x   isInherited                                  Boolean : String(propertyName)
-r-x   isSet                                        Boolean : String(propertyName)
-r-x   removeTag                                    Boolean : String(tag)
-r-x   restoreDefaultValue                          Void : String(attributeName)
-r-x   unSet                                        Void : String(propertyName)

wls:/externalstage/edit/Servers/AdminServer/ServerDebug/AdminServer/DebugScopes/weblogic.connector !> wls:/externalstage/edit/Servers/AdminServer/ServerDebug/AdminServer/DebugScopes/weblogic.connector !> set('Enabled',true)
wls:/externalstage/edit/Servers/AdminServer/ServerDebug/AdminServer/DebugScopes/weblogic.connector !> save()
Saving all your changes ...
Saved all your changes successfully.
wls:/externalstage/edit/Servers/AdminServer/ServerDebug/AdminServer/DebugScopes/weblogic.connector !> activate()
Activating all your changes, this may take a while ... 
The edit lock associated with this edit session is released once the activation is completed.
Activation completed
wls:/externalstage/edit/Servers/AdminServer/ServerDebug/AdminServer/DebugScopes/weblogic.connector> exit()

Reply:
create('weblogic.security.SSL', 'DebugScopes)